.zenodo.json

```diff
@@ -753,6 +753,9 @@
       "affiliation": "MIT, HMS",
       "name": "Ghosh, Satrajit",
       "orcid": "0000-0002-5312-6729"
+    },
+    {
+      "name": "Nat Lee"
```

Comment:
Please use the `Last, First` convention. And do you have an affiliation or ORCID you would like to associate? ([ORCID](https://orcid.org/) only take a minute to get.)

Reply:
Okay, I got it!
Thank you!